src/main/java/com/puppycrawl/tools/checkstyle/checks/design/FinalClassCheck.java

```diff
@@ -42,6 +42,46 @@
  * &lt;module name=&quot;FinalClass&quot;/&gt;
  * </pre>
  * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * final class MyClass {
+ *   private MyClass() { } //OK
+ * }
+ *
+ * class MyClass { //violation, class should be declared final
+ *   private MyClass() { }
+ * }
+ *
+ * class MyClass {
+ *   int a;
+ *   String str;
+ *   private MyClass(int a) {
+ *   this.a = a;
+ *   str = "dummy value";
```

Comment:
do it like
```
 *   int field1;
 *   String field2;
 *   private MyClass(int value) {
 *     this.field1 = value;
 *     this.field2 = "";
```

attention to indentation of code in example, and I removed extra code to let user focus on what matter - `ctors`.

Reply:
Done